servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ResponseCancelTest.java

```diff
@@ -126,54 +164,108 @@ void cancel() throws Throwable {
         // and hence fail the response.
         ClientTerminationSignal.resumeExpectFailure(delayedClientTermination, latch1,
                 instanceOf(ClosedChannelException.class));
+        clientConnectionClosed.await();
+        // serverConnectionClosed.await();  server doesn't close the connection bcz it hasn't finished the response yet
```

Comment:
consider modifying the test to verify the server cleanups too:

```diff
diff --git a/servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ResponseCancelTest.java b/servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ResponseCancelTest.java
index 97f0da293..46ddb14d6 100644
--- a/servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ResponseCancelTest.java
+++ b/servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/ResponseCancelTest.java
@@ -154,7 +154,7 @@ class ResponseCancelTest {
         CountDownLatch latch1 = new CountDownLatch(1);
         Cancellable cancellable = sendRequest(client, latch1);
         // wait for server to receive request.
-        serverResponses.take();
+        Processor<StreamingHttpResponse, StreamingHttpResponse> serverResp = serverResponses.take();
         assertThat("Unexpected connections count.", connectionCount.get(), is(1));
         cancellable.cancel();
         // wait for cancel to be observed but don't send cancel to the transport so that transport does not close the
@@ -165,9 +165,11 @@ class ResponseCancelTest {
         ClientTerminationSignal.resumeExpectFailure(delayedClientTermination, latch1,
                 instanceOf(ClosedChannelException.class));
         clientConnectionClosed.await();
-        // serverConnectionClosed.await();  server doesn't close the connection bcz it hasn't finished the response yet
 
         sendSecondRequest();
+
+        serverResp.onSuccess(client.asStreamingClient().httpResponseFactory().ok());
+        serverConnectionClosed.await();
     }
 
     @Test

```

Reply:
similar comment on other new tests